### PR TITLE
fix: Add specific error handling around git repo input

### PR DESCRIPTION
**Pull request by danmcp**
Adds error handling to return specific eval errors for known errors encountered with git repos and models missing.

This fix (with a new release of eval) must go in before: https://github.com/instructlab/instructlab/pull/1616

### CODE REVIEW

**Comment by danmcp** [comment]
Note this PR is best reviewed ignoring whitespace: https://github.com/instructlab/eval/pull/52/files?w=1

File: src/instructlab/eval/mmlu.py [hunk 1]
```diff
@@ -180,19 +204,15 @@ def run(self) -> tuple:
...
-        tm = TaskManager(verbosity="DEBUG", include_path=self.sdg_path)
 
-        mmlu_output = simple_evaluate(
-            model="hf",
-            model_args=model_args,
-            tasks=self.tasks,
-            num_fewshot=self.few_shots,
-            batch_size=self.batch_size,
-            task_manager=tm,
+        results = results = run_mmlu(
```
Comment (jaideepr97) [inline src/instructlab/eval/mmlu.py:208]: @danmcp should this just be `results = run_mmlu(...)` ?

Comment (danmcp) [reply src/instructlab/eval/mmlu.py]: Oops, yes!  Fixed.

**Comment by danmcp** [comment]
Related to #36

**Review by jaideepr97 — APPROVED** [review]
LGTM, thanks @danmcp!
Please resolve conflicts

**Comment by danmcp** [comment]
> Please resolve conflicts

Done, thanks!

**Review by alinaryan — APPROVED** [review]

**Pull request merged**